post/getting-started-preact/index.md

```diff
@@ -0,0 +1,322 @@
+---
+title: "Getting Started with Directus and Preact"
+description: "120-160 characters"
+author:
+    name: "jay bharadia"
+    avatar_file_name: "my-avatar.jpg"
+---
+
+## Introduction
+
+Fast 3kB alternative to React with the same modern API. In this tutorial, you will learn how to build a website using Directus as a headless CMS. You will store, retrieve, and use global metadata such as the site title, create new pages dynamically based on Directus items, and build a blog.
+
+Before You Start
+
+You will need:
+
+To install Node.js and a code editor on your computer.
+
+A Directus project - you can use Directus Cloud or run it yourself.
+
+Some knowledge of Preact framework
+
+## Initializing Preact
+
+The modern way to get started is using Vite bundler instead of old preact-cli
```

Comment:
This whole section is not formatted correctly, and missing loads of detail. Where's the instructions to initialize a project? Where's the installation of the Directus SDK and setup of the util file?